### ISSUE TITLE
Ready to use Project ?

### ISSUE THREAD

**Issue by RUSshy** [GameFoundry/bsfExamples#7]

**Comment by BearishSun** [comment]
Sounds like a good idea, I'll get around to it.

**Comment by cwfitzgerald** [comment]
I would agree however that it probably should be a separate repo. The question then becomes how to auto bump the versions. If autobumping could happen, then that would also be a good place to put travis-ci as you can test that you don't break api.
